contracts/ClaimsManager.sol

```diff
@@ -204,7 +213,10 @@ contract ClaimsManager is
         require(claimAmount != 0, "Claim amount zero");
         require(bytes(evidence).length != 0, "Evidence address empty");
         require(block.timestamp >= startTime, "Policy not active yet");
-        require(block.timestamp <= endTime, "Policy expired");
+        require(
+            block.timestamp <= endTime + claimValidityPeriod,
```

Comment:
Why don't we add this to the `endTime` while creating the policy? So if someone buys a policy that ends on 4th, we create it to end on 7th.

Reply:
I'm not sure I agree with this, seems more confusing than just having the condition check when a claim is submitted? For example if claimValidityPeriod is changed mid-policy, that existing policyHash would become inaccurate, and might also cause some front-end complication

Reply:
We can't (or shouldn't be able to) change policy terms mid-policy, and having a claimValidityPeriod that can be changed mid-policy exactly allows that. Compared to that, a policy created to end 3 days later than it should enforces an immutable claimValidityPeriod.
Also, adding claimValidityPeriod to endTime at policy creation means changing claimValidityPeriod won't require an additional transaction (you just create the future policies with the new claimValidityPeriod), which is slightly more convenient.

Reply:
Ok good points, I've made that change. 
I also added `endTime += claimValidityPeriod;` to createClaim() since presumably the claim submitter will know their policy end time but shouldn't be expected to add on the extra claimValidityPeriod in order for the policyHash to match upon submission. Alternatively we could have the claim submitter enter their endTime _including_ claimValidityPeriod when submitting a claim (they could get it from createPolicy()'s CreatedPolicy event) but I figured the former is simpler -- let me know if you disagree.

Reply:
I agree that the current implementation is incorrect (we will have to enter a policy end time that's not equal to what says in the PDF, and this may even be used against us in the dispute to make a claim for an event past the policy end time), but a very easy solution is to not specify "policy end time" but rather "claim validity end time" (though "validity" is probably a bit opinionated, "permission" may be better) while creating the policy (so a bit of variable renaming). The issue with this PR is that it stores information on-chain that's not used by any contract logic (the contract doesn't care about the policy end time but only claim validity period end time, the jurors need to know the policy end time and they will refer to the PDF for that, not contract/events). I see your point of "represent things that can be represented in the contract implementation", but this only applies to processes, not data.

As a note, you're mutating the endTime variable, which causes its name to be incorrect at some point. For example this
```
function(endTime) {
  endTime += claimValidityPeriod
}
```
should probably look something like this instead
```
function(policyEndTime) {
  claimValidityPeriodEndTime = policyEndTime + claimValidityPeriod
}
```

Reply:
I've now made the variable name changes (endTime to claimValidityPeriodEndTime), including in IClaimsManager.sol - I think "validity" is proper since it refers to the objective timeliness of the claim submission. 

A lingering question is whether we should preserve setClaimValidityPeriod (function and event), as it's no longer used for anything directly here; I suppose the argument for preserving it would be an on-chain memorialization of a claim validity period change or usage by any inheriting contracts?

Reply:
It's confusing to keep redundant functionality so I removed it